[DO NOT MERGE] feat: Build Pricing Table with Azion Edge SQL (#888)
aziontech/docs: src/content/docs/en/pages/main-menu/pricing/pricing.mdx

    'USA',
    'Canada',
    'Europe',
    'Brazil',
    'Latam',
    'All Other Regions'
  ]}
  product_slug="content_delivery"
  metric_slug="data_transferred"
/>

<TablePricing
  lang="en"
  metric='Requests | per 10,000'
  billing={[
    'USA',
    'Canada',
    'Europe',
    'Brazil',
    'Latam',
    'All Other Regions'
  ]}
  product_slug="content_delivery"
  metric_slug="cdn_requests"
/>

#### Price example
If your applications received 750 million requests from the United States in 1 month, consuming a total traffic of 30 TB, and you were within the limit of 1,000 purges in that month, the price will be calculated as follows:

|  | |
| :--- | ---: |
| **Total Data Transferred**<br/>The first 10 TB of traffic is priced at 0.085 USD per GB, so 10,000 x 0.085 USD | 850 USD | 
| The next 20 TB of traffic is priced at 0.080 USD per GB, so 20,000 x 0.080 USD | 1,600 USD | 
| In this example, the price for this month will be | 2,450 USD | 
| The charge is for every 10,000 requests, at a unit price of 0.009 USD, so (750,000,000 / 10,000) x 0.009 USD | 675 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **3,125 USD** | 

> 

Reviewer: ```suggestion
```

### Application Accelerator

Speed ​​up the performance of your applications and APIs with protocol optimizations and advanced request and response rules.

> <Button href="https://www.azion.com/en/products/application-acceleration/" text="Access product details" variant="secondary"></Button>

Reviewer: ```suggestion
<Button href="https://www.azion.com/en/products/application-accelerator/" text="Access product details" variant="secondary"></Button>
```

| The remaining 500 million will be priced at 0.40 USD per million. So 500 x 0.40 USD | 200 USD | 
| In this example, the calculated value will be | 1,050 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **1,750 USD** |

---

## Deploy

### Edge Orchestrator

Deploy, control, automate, and monitor features on the edge, in real time with Azion Edge Orchestrator.

<Button href="https://www.azion.com/en/products/edge-orchestrator/" text="Access product details" variant="secondary"></Button>

---

## Observe

### Data Stream

Get real-time data access to power your big data, stream processing, SIEM, and serverless application platforms.

<Button href="https://www.azion.com/en/products/data-stream/" text="Access product details" variant="secondary"></Button>

<TablePricing
  lang="en"
  metric='Data Stream | Requests'
  billing={[
    'Price per 10,000'
  ]}
  product_slug="data_streaming"
  metric_slug="data_streaming_requests"
/>

<TablePricing
  lang="en"
  metric='Data Streamed'
  billing={[
    'Price per GB'
  ]}
  product_slug="data_streaming"
  metric_slug="data_streaming_data_streamed"
/>

#### Price example

> If you had 3 billion **Data Stream** requests and streamed 25TB of data, the value would be calculated as follows:

Reviewer: ```suggestion
If you had 3 billion **Data Stream** requests and streamed 25 TB of data, the value would be calculated as follows:
```

| --- | ----------- |
| First 1,000<br/>queries/month | Included |
| Over 1,000<br/>queries/month | 0.10 USD |

#### Price example

If your applications have 300 TB of traffic in a month, most of that volume is data, such as payload, while other traffic is metadata, such as HTTP headers, the request made, status, or user information. Real-Time Events collects some of the metadata, which are made available for your query. The metadata collected represents on average *0.2% of your traffic*.

| | |
| :--- | ---: |
| In this simulation, 600 GB (300 TB x 0.2%) will be gathered. |  | 
| The first 400 GB will be charged at a unit price of 1.50 USD, so 400 x 1.50 USD | 600 USD | 
| The remaining 200 GB will be charged at a unit price of 1.20 USD, so 200 x 1.20 USD | 240 USD |
| **Total charges**<br/>Considering that the limit of 1,000 queries isn't exceeded, the total price calculated in this simulation will be | **840 USD** |

:::note[Queries]
This is the total number of **Real-Time Events** queries. The first 1,000 queries in a month are included for all clients. When exceeding this volume, you'll be charged 0.10 USD per query.
:::

---

## Azion Service Plans

### Developer Plan

> For developers who want to explore the potential of our platform themselves.

Reviewer: ```suggestion
For developers who want to explore the potential of Azion's platform themselves.
```

meta_tags: edge computing, azion, pricing
namespace: documentation_pricing
permalink: /documentation/products/pricing/
---

import Button from '~/components/Button.astro'
import Badge from '~/components/Badge.astro';
import TablePricing from '~/components/Table/TablePricing.astro';

This documentation includes prices for Azion products, services, and Service Plans.

---

## Build

### Edge Application

Build serverless applications and run them anywhere: Azion's—operated network edge, remote-devices, on-premises, and multi-cloud, on an easy and simple way.

<Button href="https://www.azion.com/en/products/edge-application/" text="Access product details" variant="secondary"></Button>

<TablePricing
  lang="en"
  metric='Total Data Transferred | per GB'
  billing={[
    'USA',
    'Canada',
    'Europe',
    'Brazil',
    'Latam',
    'All Other Regions'
  ]}
  product_slug="content_delivery"
  metric_slug="data_transferred"
/>

<TablePricing
  lang="en"
  metric='Requests | per 10,000'
  billing={[
    'USA',
    'Canada',
    'Europe',
    'Brazil',
    'Latam',
    'All Other Regions'
  ]}
  product_slug="content_delivery"
  metric_slug="cdn_requests"
/>

#### Price example

> If your applications received 750 million requests from the United States in 1 month,
> consuming a total traffic of 30TB, and you were within the limit of 1,000 purges in that month, the price will be calculated as follows:

Reviewer: ```suggestion
If your applications received 750 million requests from the United States in 1 month, consuming a total traffic of 30 TB, and you were within the limit of 1,000 purges in that month, the price will be calculated as follows:
```

|  | |
| :--- | ---: |

> | **Total Data Transferred**<br/>The first 10TB of traffic is priced at 0.085 USD per GB, so 10,000 x 0.085 USD | 850 USD | 
> | The next 20TB of traffic is priced at 0.080 USD per GB, so 20,000 x 0.080 USD | 1,600 USD | 
> | In this example, the price for this month will be | 2,450 USD | 
> | The charge is for every 10,000 requests, at a unit price of 0.009 USD, so (750,000,000 / 10,000) x 0.009 USD | 675 USD | 
> | **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **3,125 USD** | 

Reviewer: ```suggestion
| **Total Data Transferred**<br/>The first 10 TB of traffic is priced at 0.085 USD per GB, so 10,000 x 0.085 USD | 850 USD | 
| The next 20 TB of traffic is priced at 0.080 USD per GB, so 20,000 x 0.080 USD | 1,600 USD | 
| In this example, the price for this month will be | 2,450 USD | 
| The charge is for every 10,000 requests, at a unit price of 0.009 USD, so (750,000,000 / 10,000) x 0.009 USD | 675 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **3,125 USD** | 
```

### Application Accelerator

Speed ​​up the performance of your applications and APIs with protocol optimizations and advanced request and response rules.

<Button href="https://www.azion.com/en/products/application-acceleration/" text="Access product details" variant="secondary"></Button>

<TablePricing
  lang="en"
  metric="Data Transferred | per GB"
  billing={[
    'USA',
    'Canada',
    'Europe',
    'Brazil',
    'Latam',
    'All Other Regions'
  ]}
  product_slug="application_acceleration"
  metric_slug="application_acceleration_data_transferred"
/>

#### Price example

> 

Reviewer: ```suggestion
```

> If your **Application Accelerator** enabled application generated 30TB of data transferred in Brazil in 1 month, the price will be calculated as follows:

Reviewer: ```suggestion
If your **Application Accelerator** enabled application generated 30 TB of data transferred in Brazil in 1 month, the price will be calculated as follows:
```

| | |
| :--- | ---: |

> | The first 10TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
> | The next 20TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
> | **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **1,100 USD** | 

Reviewer: ```suggestion
| The first 10 TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
| The next 20 TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **1,100 USD** | 
```

| | |
| :--- | ---: |
| The first 10 million images processed will be priced at 1.60 USD for every 10,000 images, for example: (10,000,000 / 10,000) x 1.60 USD | 1,600 USD | 
| The next 20 million images processed will be priced at 1.28 USD for every 10,000 images, for example: (20,000,000 / 10,000) x 1.28 USD | 2,560 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **4,160 USD** |

### Load Balancer

Balance the load on your origin and improve the reliability and availability of your content, applications, and APIs.

<Button href="https://www.azion.com/en/products/load-balancer/" text="Access product details" variant="secondary"></Button>

<TablePricing
  lang="en"
  metric='Data Transferred | per GB'
  billing={[
    'USA',
    'Canada',
    'Europe',
    'Brazil',
    'Latam',
    'All Other Regions'
  ]}
  product_slug="load_balancer"
  metric_slug="load_balancer_data_transferred"
/>

#### Price example

> If your **Load Balancer** enabled application generated 30TB of data transferred in Brazil in 1 month, the price will be calculated as follows:
> 
> | | |
> | :--- | ---: |
> | The first 10TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
> | The next 20TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
> | **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **1,100 USD** |

Reviewer: ```suggestion
If your **Load Balancer** enabled application generated 30 TB of data transferred in Brazil in 1 month, the price will be calculated as follows:

| | |
| :--- | ---: |
| The first 10 TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
| The next 20 TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **1,100 USD** |
```

Free of charge
</Badge>

Monitor the user experience when accessing your applications and improve network availability and performance.

<Button href="https://www.azion.com/en/products/edge-pulse/" text="Access product details" variant="secondary"></Button>

### Real-Time Metrics

<Badge variant="accent">
Free of charge
</Badge>

Powerful insight into your applications' performance, availability, and security in real time.

<Button href="https://www.azion.com/en/products/real-time-metrics/" text="Access product details" variant="secondary"></Button>

### Real-Time Events

Explore the data generated by your applications on Azion's platform and perform complex queries to track what is happening with your applications in real time.

<Button href="https://www.azion.com/en/products/real-time-events/" text="Access product details" variant="secondary"></Button>

| Data Gathered | Price per GB |
| --- | ----------- |
| First 400<br/>GB/month | 1.50 USD |
| Next 600<br/>GB/month | 1.20 USD |
| Over 1<br/>TB/month | 0.60 USD |

| Queries | Unitary |
| --- | ----------- |
| First 1,000<br/>queries/month | Included |
| Over 1,000<br/>queries/month | 0.10 USD |

#### Price example

> If your applications have 300TB of traffic in a month, most of that volume is data, such as payload, while other traffic is metadata, such as HTTP headers, the request made, status, or user information. Real-Time Events collects some of the metadata, which are made available for your query. The metadata collected represents on average *0.2% of your traffic*.

Reviewer: ```suggestion
If your applications have 300 TB of traffic in a month, most of that volume is data, such as payload, while other traffic is metadata, such as HTTP headers, the request made, status, or user information. Real-Time Events collects some of the metadata, which are made available for your query. The metadata collected represents on average *0.2% of your traffic*.
```

| | |
| :--- | ---: |

> | In this simulation, 600GB (300 TB x 0.2%) will be gathered. |  | 
> | The first 400GB will be charged at a unit price of 1.50 USD, so 400 x 1.50 USD | 600 USD | 
> | The remaining 200GB will be charged at a unit price of 1.20 USD, so 200 x 1.20 USD | 240 USD |
> | **Total charges**<br/>Considering that the limit of 1,000 queries isn't exceeded, the total price calculated in this simulation will be | **840 USD** |

Reviewer: ```suggestion
| In this simulation, 600 GB (300 TB x 0.2%) will be gathered. |  | 
| The first 400 GB will be charged at a unit price of 1.50 USD, so 400 x 1.50 USD | 600 USD | 
| The remaining 200 GB will be charged at a unit price of 1.20 USD, so 200 x 1.20 USD | 240 USD |
| **Total charges**<br/>Considering that the limit of 1,000 queries isn't exceeded, the total price calculated in this simulation will be | **840 USD** |
```
